Source/RageCommunity.Library/Wrappers/NativeTaskWrappers.cs

```diff
@@ -13,10 +13,24 @@ public static partial class NativeWrappers
         /// <remarks>
         /// A list of scenarios can be found <a href="https://github.com/DurtyFree/gta-v-data-dumps/blob/master/scenariosCompact.json">here</a>.
         /// </remarks>
-        public static void TaskStartScenarioInPlace(Ped ped, Scenario scenario, int unkDelay = 0, bool playEnterAnimation = true)
+        public static void TaskStartScenarioInPlace(Ped ped, string scenario, int unkDelay = 0, bool playEnterAnimation = true)
```

Comment:
Why did you change this from the Scenario enum?

Reply:
because the [natives](https://nativedb.dotindustries.dev/natives/0x142A02425FF02BD9) says that this parameter is a string

Reply:
I added a change suggestion to convert the parameter to a string on the native this wrapper points to.  Using the enum in the wrapper can help the user through intellisense.